src/main/java/codeu/model/store/basic/ActivityStore.java

```diff
@@ -27,16 +27,17 @@ public static ActivityStore getInstance() {
 
   // In memory-queue or recent activities
   private LinkedList<Activity> activities;
-  private final static int maxNumActivities = 5;  //FIXME to 100
+  private final static int maxNumActivities = 100;  
 
   private ActivityStore(PersistentStorageAgent persistentStorageAgent) {
     this.persistentStorageAgent = persistentStorageAgent;
     this.activities = new LinkedList<Activity>();  
   }
 
   public void addActivity(Activity newActivity) {
-    if(activities.size() > maxNumActivities) {
+    if(activities.size() >= maxNumActivities) {
       activities.removeFirst();  //full queue, remove oldest
+      System.err.println("THERE SHOULD BE ONE LEST ACTCITIY NOW, is " + activities.size());
```

Comment:
Was this line added for testing/debugging purposes? I am not sure if this is needed for the actual web page.